modules/ROOT/pages/digger/access-mobile-client-build-logs.adoc

```diff
@@ -0,0 +1,22 @@
+[[access-mobile-client-build-logs]]
+= Viewing Build Logs
+
+== Viewing Build History in the OpenShift Console
+
+All of the builds related to your mobile client can be seen in the **Builds** tab of your {mobile-client}. Each build can be expanded to show further information about the build configuration, latest build and build history.
+
+The latest build, as well as previous builds, has a `View Log` link associated with them to access the detailed logs of that build.
+
+image::mobile-clients-builds-complete.png[access-build-from-notification][align="center"]
+
+
+== Viewing Build Logs in Jenkins
+
+. Click the `View Log` link for the build you want to view logs, you are redirected to the Jenkins instance that has been running your build.
```

Comment:
Choose a build log and click `View Log`, you are redirected to the Jenkins instance that has been running your build.